tests/integration-tests/configs/develop.yaml

```diff
@@ -266,28 +266,6 @@ test-suites:
           instances: ["c5n.18xlarge"]
           oss: ["centos7"]
           schedulers: ["slurm"]
-        - regions: ["use1-az6"]   # do not move, unless capacity reservation is moved as well
```

Comment:
I feel like running with p4d is better as historically we have had multiple issues with EFA tests or other job failures on these intance types

Reply:
Everything else except this looks good to me.